.prettierrc

```
@@ -0,0 +1 @@
+'trailingComma': 'none'
```

```suggestion
'trailingComma': 'es5'
```

(this would reduce the diff, if we go the trailingComma option)

---

Then we need to agree n a new convention that we work with trailing , - @mouse-reeve, what do you say?